compositor_render/src/renderer/scene.rs

```diff
@@ -49,6 +48,19 @@ impl TransformNode {
                     ctx.wgpu_ctx,
                     ctx.shader_transforms.get(shader_id)?,
                     shader_params.as_ref(),
+                    None,
+                )),
+                *resolution,
+            )),
+            TransformParams::Builtin {
+                transformation,
+                resolution,
+            } => Ok((
+                TransformNode::Common(ShaderNode::new(
```

Comment:
I susspect that doing just wrapper around ShaderNode will not be enough for long, but on the other hand solutions written this way are easy to copy-paste by user and modify them for their own usecases

Reply:
When we'll need to use different transforms here, we can always wrap it in some struct.
Since there aren't any transformations so far, that couldn't be implemented as shaders, I would leave it as it is right now

Reply:
I think with some abstraction layer over the shader pipeline (single shader transformation pipeline), we could make a few things more straightforward / more efficient and don't use ShaderNode here (e.g. calculate matrixes on CPU and send them via uniform), but I would do it here when it's useful.